api/src/lib.rs

```diff
@@ -328,6 +328,7 @@ where
             };
             let vb_new = serde_json::to_vec(&vault).map_err(server_err)?;
             db.put(&id, &vb_new).await.map_err(server_err)?;
+            db.del_version("v1", id).await.map_err(server_err)?;
```

Comment:
this `v1` isn't related to our API version, right?

Reply:
this removes the old vault data once it's been migrated. it's kinda related. why do you ask?

Reply:
That makes sense, I was just worried that it could mean the delete action wouldn't work in v2